ratpack-core/src/main/java/ratpack/http/client/RequestSpec.java

```diff
@@ -117,6 +117,12 @@ default RequestSpec sslContext(Factory<SSLContext> factory) throws Exception {
 
   URI getUrl();
 
+  default RequestSpec connectTimeoutSeconds(int seconds) {
```

Comment:
I can confirm that the default connect timeout is 30 seconds. Would you like me to make the change to default to that, removing the Optional?